packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -310,6 +310,43 @@ export const defaultL2Network: ArbitrumNetwork = {
   }
 }
 
+// ETH as native token
```

Comment:
This will be added back before merging and the non18 decimal chain config will be commented out. We can think of a better solution for this in the future